src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -569,6 +569,13 @@ class Testcase(Model):
   # Uploader email address.
   uploader_email = ndb.StringProperty()
 
+  # Github repository id
+  github_repo_id = ndb.IntegerProperty()
+
+  # Github issue number.
```

Comment:
nit: this comment isn't useful, what it is saying can be understood by reading the variable name.

Reply:
Fixed in [More informative comments](https://github.com/google/clusterfuzz/pull/2578/commits/e5f266fa5586ee43ab8ac75d8030e3b37869a523).